### PR TITLE
Monorepo Support

### LINKED ISSUE #14: Deploy to monorepo subfolders
I have a monorepo containing a number of react components that i've built using storybook (IT'S AWESOME!).

I would like to generate GH Pages for each of them, however storybook-deployer seems to always deploy to the root of the mono repo for GH Pages instead of the currently active folder.

### PR DESCRIPTION
**Pull request by hipstersmoothie**
Adds support for monorepos.

Accomplished by:

- Adding `packages` flag that specifies a folder that sub-packages are kept in
- Use [glob](https://www.npmjs.com/package/glob) to find all `package.json`s (excluding `node_modules`)
- Go to the directory, make sure its has a `.storybook` directory or specifies the build script and built it
- Move the built storybook to the root output directory and clean up
- Continue as normal

NOTE: I have tested this on my own monorepo storybook and it works. It is on a private Github though, so I cannot share the results.

closes #14

### CODE REVIEW

**Comment by hipstersmoothie** [comment]
Hey @ndelangen could you review this? Thanks!

**Comment by hipstersmoothie** [comment]
@igor-dv @Hypnosphi ?

**Comment by igor-dv** [comment]
Sorry I wasn't in focus of the storybook-deployer. Will check this today.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,16 @@ If you have previously built your storybook output (through a different CI step,
 npm run deploy-storybook -- --existing-output-dir=.out
 ```
 
+### Deploy a Monorepo
+
+If you manage a monorepo with multiple storybooks your can you pass the `packages` flag to `deploy-storybook` to scan a directory for `package.json`s.
```
Comment (igor-dv) [inline README.md]: `your can you pass` => `you can pass`

**Pull request merged**